servers/src/grin/server.rs

```diff
@@ -506,17 +498,16 @@ impl Server {
 			total_difficulty: head.total_difficulty(),
 		};
 
-		let header_stats = match self.chain.try_header_head()? {
-			Some(tip) => {
-				let header = self.chain.get_block_header(&tip.hash())?;
-				Some(ChainStats {
-					latest_timestamp: header.timestamp,
-					height: header.height,
-					last_block_h: header.prev_hash,
-					total_difficulty: header.total_difficulty(),
-				})
-			}
-			_ => None,
+		let header_stats = if let Some(head) = self.chain.try_header_head()? {
```

Comment:
I personally find `match` more readable than exhaustive `if let /else`, but it's a matter of taste

Reply:
Yeah maybe I'll revert this back to `match`. I'm not really sure which I prefer in cases like this.

Reply:
Now using `match` and a `map` like we do for the tx_stats above. I think this reads better now.